### PR TITLE
Bump default PG conns from 20->100; enable auto-scaling open conns for mercury

### PR DESCRIPTION
**Pull request by samsondav**

### CODE REVIEW

File: core/services/pg/connection.go [hunk 1]
```diff
@@ -94,6 +93,33 @@ func NewConnection(uri string, dialect dialects.DialectName, config ConnectionCo
...
+	// This problem will be solved by replacing mercury with parallel
+	// compositions (llo plugin).
+	//
+	// See: https://smartcontract-it.atlassian.net/browse/MERC-3654
+	var cnt int
+	if err := db.Get(&cnt, `SELECT COUNT(*) FROM ocr2_oracle_specs WHERE plugin_type = 'mercury'`); err != nil {
+		log.Printf("Error checking mercury jobs: %s", err.Error())
+		return
+	}
+	if cnt > config.MaxOpenConns() {
```
Comment (Fletch153) [inline core/services/pg/connection.go:113]: I think this is a good solution until parallel comp, but but 1 job per connection might cause issues if we have 1000 jobs

**Review by Fletch153 — DISMISSED** [review]

File: core/services/pg/connection.go (hunk 1 above)
Comment (samsondav) [reply core/services/pg/connection.go:113]: We can't run that many jobs on one node performantly then, would need to have multiple DONs

**Review by RensR — CHANGES_REQUESTED** [review]
please undo the forge-std change

**Review by jmank88 — APPROVED** [review]

**Review by RensR — APPROVED** [review]
just undoing the requested changes

**Pull request merged**